deploy/helm/sumologic/values.yaml

```diff
@@ -1123,7 +1124,8 @@ kube-prometheus-stack:
     ## Put here the new name if you want to override the full name used for Prometheus Node exporter components.
     # fullnameOverride: ""
 
-    nodeSelector: {}
+    nodeSelector:
+      kubernetes.io/os: linux
```

Comment:
This configuration for prometheus node exporter and we should explicitly leave it as here